### PR TITLE
[DEVHAS-736] Create techdoc for running applications in RHOAI

### PR DESCRIPTION
**Pull request by johnmcollier**
Fixes https://issues.redhat.com/browse/DEVHAS-736

Implements a techdoc for running the sample apps in OpenShift AI:
-  https://github.com/redhat-ai-dev/ai-lab-template/blob/76fa5d59ad4e4aac999268ff9dd4b58fa1081b8a/skeleton/techdoc/docs/rhoai.md if you want to see the templated markdown file
- Includes instructions on building and running sample apps locally, as well as how to port-forward the sample app's port to the user's local machine
- I wasn't sure how to conditionally show the techdoc when RHOAI was selected, so I just prefaced it with saying that RHOAI must have been selected during component creation
- I included a couple images in the techdoc, but it seems that images can't be displayed in techdocs

### CODE REVIEW

File: skeleton/techdoc/docs/rhoai.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Running Samples in OpenShift AI
+
+This document will outline how you can run build and run your sample applications within an OpenShfit AI workbench.
```
Comment (Jdubrick) [inline skeleton/techdoc/docs/rhoai.md:3]:
```suggestion
This document will outline how you can run build and run your sample applications within an OpenShift AI workbench.
```

Comment (Jdubrick) [inline skeleton/techdoc/docs/rhoai.md:3]:
```suggestion
This document will outline how you can run build and run your sample applications within an OpenShift AI workbench.
```

File: skeleton/techdoc/docs/rhoai.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+## Prerequisites
+
+- `oc` cli installed
```
Comment (Jdubrick) [inline skeleton/techdoc/docs/rhoai.md:7]: Should we link to `oc` install page here?

**Review by Jdubrick — CHANGES_REQUESTED** [review]

File: skeleton/techdoc/docs/rhoai.md (hunk 2 above)
Comment (johnmcollier) [reply skeleton/techdoc/docs/rhoai.md:7]: Yeah, that's a good idea. I'll link https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/stable/openshift-client-mac.tar.gz which will provide the latest stable binary

Comment (johnmcollier) [reply skeleton/techdoc/docs/rhoai.md:8]: Updated

**Comment by johnmcollier** [comment]
@Jdubrick Review comments addressed if you don't mind re-reviewing.

**Review by Jdubrick — APPROVED** [review]
lgtm

**Pull request merged**